mobile/examples/ort_extensions_react_native_usage/ort-ext-rn-usage/README.md

```diff
@@ -0,0 +1,105 @@
+# ONNX Runtime React Native Sample Usage Application with ort-extensions
+
+## Overview
+
+This is a basic example usage react native expo application for [ONNX Runtime](https://github.com/microsoft/onnxruntime) with [Ort-Extensions](https://github.com/microsoft/onnxruntime-extensions) support for pre/post processing. The demo app accomplishes the task of loading a model with pre/post processing ops and run inference session to get the output result.
+
+0. Prepare environment
+    1. install Node.js
+    2. install expo
+
+        ```sh
+        npm install -g expo-cli
+        ```
+
+    3. install yarn
+
+        ```sh
+        npm install -g yarn
+        ```
+
+1. Setup empty project
+
+   ```sh
+   cd <SOURCE_ROOT>
+   expo init . -t expo-template-blank-typescript
+   yarn
+   ```
+
+   **NOTE:**
+   - `<SOURCE_ROOT>` refers to the root folder of the source code, where this `README.md` file sits.
+   i.e. `/mobile/examples/ort_extensions_react_native_usage/ort-ext-rn-usage`
+
+2. Install onnxruntime-react-native
+
+    ```sh
+    expo install onnxruntime-react-native@dev
```

Comment:
is "@dev" meant to be?

Reply:
yes, add @dev will automatically install the latest dev version onnxruntime-react-native npm package.